spec/requests/projects_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+RSpec.describe Project, type: :request do
+  let!(:user) { create(:user) }
+  let(:token) { JsonWebToken.encode(user_id: user.id) }
+  let!(:project) { create(:project, user_id: user.id) }
```

Comment:
what the purpose of this exact let? Could you please remove.

`let` with bang slows down specs. It is not big deal in such little project like TodoAPI application, but better avoid it.

Also, if you need database to be pre-filled with some entitites, you can:
1) just make something like this:
```ruby
before do
  create(:project)
end
```
2) use gem `test-prof`, which provides `let_it_be` helper method, which is the same as `let!`, but optimized